### PR TITLE
Normalize mardkown formatting across usage.md files

### PR DESCRIPTION
**Pull request by ascott1**
Bringing #292 to release so that these changes can be incorporated into the documentation.

### CODE REVIEW

**Comment by Scotchester** [comment]
In this PR, you need to increment each component's version number by one patch level.

**Comment by ascott1** [comment]
@Scotchester do I do that locally and push directly to the `canary` branch?

**Comment by Scotchester** [comment]
Yes, exactly. I think. Let me double-check.

**Comment by Scotchester** [comment]
Yes, that's right.

**Comment by ascott1** [comment]
Done.

File: src/cf-pagination/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cf-pagination",
-  "version": "3.0.3",
+  "version": "3.0.8",
```
Comment (Scotchester) [inline src/cf-pagination/package.json]: Should be 3.0.4.

Comment (ascott1) [inline src/cf-pagination/package.json]: Argh, whoops.

Comment (ascott1) [inline src/cf-pagination/package.json]: Corrected

**Comment by Scotchester** [comment]
:+1:

**Comment by Scotchester** [comment]
Feel free to merge after Travis passes its checks.

**Comment by ascott1** [comment]
:+1: Thanks for all of your help @Scotchester!

**Pull request merged**